pkg/client/client.go

```diff
@@ -93,15 +119,49 @@ func New(opts command.Options) (*Client, error) {
 
 	c.paginationManager = pm
 
-	return &c, nil
+	return c, nil
+}
+
+func (c *Client) SetActiveDatabase(database string) {
+	c.activeDatabase = database
+}
+
+func (c *Client) ActiveDatabase() string {
+	return c.activeDatabase
+}
+
+// DB Return the db attribute.
+func (c *Client) DB() *sqlx.DB {
+	return c.db
+}
+
+// Driver returns the driver of the database.
+func (c *Client) Driver() string {
+	return c.driver
+}
+
+func (c *Client) ShowDataCatalog() bool {
+	return c.showDataCatalog
 }
 
 // Query returns performs the query and returns the result set and the column names.
 func (c *Client) Query(q string, args ...interface{}) ([][]string, []string, error) {
-	resultSet := [][]string{}
+	var (
+		resultSet = [][]string{}
+		db        *sqlx.DB
+	)
+
+	if c.activeDatabase != "" {
+		switch c.driver {
+		case drivers.Postgres, drivers.PostgreSQL, drivers.MySQL:
+			db = c.dbs[c.activeDatabase]
```

Comment:
What about this?


```suggestion
			db, ok = c.dbs[c.activeDatabase]
			if !ok {
	 			return nil, nil, return nil, fmt.Errorf("connection with %s database not found", c.activeDatabase)
			}
```

I'm afraid about possible panic, when calling db.Query

Reply:
Yeah, good catch there!

I'll add this.

Thanks!